CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+project(nerlnet DESCRIPTION "Nerlnet Distributed Machine Learning Research Platform")
+cmake_minimum_required(VERSION 3.16.3 FATAL_ERROR)
+
+find_package(OpenMP REQUIRED)
+
+set(PROJECT_BINARY_DIR build)
+set (CMAKE_CXX_STANDARD 11)
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fopenmp")#${OpenMP_C_FLAGS}")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fopenmp")#${OpenMP_CXX_FLAGS}")
```

Comment:
-latomic should be added to support Raspberry pi compilation @kapelnik @evgenyan95

Reply:
Done
https://github.com/leondavi/NErlNet/pull/69/commits/2867870d70cac221a920697aea81bcc2fdedd91d